6001-7000/LIT6990Taraf.xml

```diff
@@ -22,7 +23,7 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                 </availability>
             </publicationStmt>
             <sourceDesc>
-                <listWit><witness corresp="BLorient492#a1"> <!--<title ref="#t2"></title>--></witness></listWit>
+                <listWit><witness corresp="BLorient492#a1"/></listWit>
```

Comment:
here only the "clean" manuscript reference, if you want to specify the item write it as text 
`<witness corresp="BLorient492">BL oriental 492, as <ref target="BLorient492#a1">additio 1</ref></witness>`